lib/vsc/install/shared_setup.py

```diff
@@ -662,7 +679,10 @@ def make_release_tree(self, base_dir, files):
                     if pyshebang_reg.search(first_line):
                         log.info("going to adapt shebang for script %s" % fn)
                         dest, code = self._recopy(base_dir, fn)
-                        code = pyshebang_reg.sub(SHEBANG_STRIPPED_ENV_PYTHON, code)
+                        pyshebang_line = first_line.split()
```

Comment:
what is this doing?

Reply:
Instead of replacing the full line with `SHEBANG_STRIPPED_ENV_PYTHON`, this only replaces the call in the shebang (the first element) with  `SHEBANG_STRIPPED_ENV_PYTHON`. Hence, any arguments in the shebang are preserved.

Reply:
can't you fix the `pyshebang_reg` so the `.sub` only replaces that part?

Reply:
That would require changing `SHEBANG_ENV_PYTHON` to `'#!/usr/bin/env`, is that ok?

Reply:
Or alternatively, create a new global